.github/hooks/pre-push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "Running ESLint & Karma"
+
+npm run lint
+lint=$?
+if [ $lint -ne 0 ]; then
+    echo "ESLint Failed. Use --no-verify to force push. Please don't. \n\n\n\n"
```

Comment:
Use `--no-verify` or `-n`? Also, if we don't want people to do this, should we even include it?